packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -276,46 +286,17 @@ contract SphinxUtils is SphinxConstants {
             ISphinxScript(_script).configureSphinx();
         }
 
-        SphinxConfig memory config = ISphinxScript(_script).sphinxFetchConfig();
+        UserSphinxConfig memory config = ISphinxScript(_script).sphinxFetchConfig();
         validate(config);
         return config;
     }
 
-    function assertValidOwners(SphinxConfig memory _config) private pure {
```

Comment:
We no longer need this check since we do all of this validation when the project is created via the website.